### PR TITLE
Add responsive header

### PR DESCRIPTION
**Pull request by ogonekmateusz**
image
image

### CODE REVIEW

File: src/components/HeaderView.vue [hunk 1]
```diff
@@ -1,32 +1,32 @@
...
-              <li><a class="dropdown-item" href="#">Something else here</a></li>
-            </ul>
+          <li class="nav-item">
+            <RouterLink class="nav-link" to="/">About us</RouterLink>
+          </li>
+          <li class="nav-item">
+            <RouterLink class="nav-link " to="/">Blog</RouterLink>
+          </li>
+          <li class="nav-item ">
+            <RouterLink class="nav-link" to="/">Contact</RouterLink>
```
Comment (michalzajac123) [inline src/components/HeaderView.vue]: Dodaj style do contact

Comment (ogonekmateusz) [reply src/components/HeaderView.vue]: oki

**Review by michalzajac123 — CHANGES_REQUESTED** [review]
Dodaj style do contact

**Review by michalzajac123 — APPROVED** [review]
Good

**Pull request merged**